src/UI/Settings/MediaManagement/Sorting/SortingViewTemplate.hbs

```diff
@@ -2,6 +2,29 @@
     <legend>Folders</legend>
 
     <div class="form-group">
+        <label class="col-sm-3 control-label">Update series if folder is missing</label>
+
+        <div class="col-sm-9">
+            <div class="input-group">
+                <label class="checkbox toggle well">
+                    <input type="checkbox" name="updateSeriesIfFolderMissing"/>
+
+                    <p>
+                        <span>Yes</span>
+                        <span>No</span>
+                    </p>
+
+                    <div class="btn btn-primary slide-button"/>
+                </label>
+
+                <span class="help-inline-checkbox">
+                    <i class="icon-sonarr-form-info" title="Update series info for existing episodes even if the series folder is missing during disk scan"/>
```

Comment:
Again, update is the wrong term here. It may be moot if we decide the foot folder existing and not being empty is enough safety though.